src/stratis_cli/_error_reporting.py

```diff
@@ -169,7 +185,7 @@ def _interpret_errors(errors):
             # pylint: disable=bad-continuation
             isinstance(error, dbus.exceptions.DBusException)
             and error.get_dbus_name() == "org.freedesktop.DBus.Error.NoReply"
-        ):
+        ):  # pragma: no cover
```

Comment:
Put a FIXME here, too. We could probably test this by setting the timeout environment variable extremely low.